server/game/cards/03_TWI/events/SelfDestruct.ts

```diff
@@ -0,0 +1,33 @@
+import { EventCard } from '../../../core/card/EventCard';
+import AbilityHelper from '../../../AbilityHelper';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class SelfDestruct extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '8060312086',
+            internalName: 'selfdestruct'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Defeat a friendly unit. If you do, deal 4 damage to a unit.',
```

Comment:
I am having people not include the "If you do" part in the title and I plan to update things so that the `ifYouDo` title will be automatically added to the ability name